### PR TITLE
HARVESTER: prometheus configuration

### PR DESCRIPTION
**Pull request by DaiYuzeng**
## Linked Issues

- [https://github.com/harvester/harvester/issues/1531](https://github.com/harvester/harvester/issues/1531)

#### setting list
截屏2022-02-25 上午10 08 40

#### configuration
截屏2022-02-28 上午10 58 16

#### tips
截屏2022-02-28 上午11 01 50

截屏2022-02-28 上午11 01 57

#### default value
截屏2022-02-28 上午11 00 27

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
Both the design and typo is LGTM.

File: models/harvester/management.cattle.io.managedchart.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        ...this.listLocation.params,
+        resource: HCI.SETTING
+      }
+    };
+  }
+
+  get customValue() {
+    const spec = this.spec.values.prometheus.prometheusSpec;
+
+    return spec ? JSON.stringify(spec) : '';
```
[the remark this answers is not in the document]
Comment (DaiYuzeng) [reply models/harvester/management.cattle.io.managedchart.js]: Done.

**Review by WuJun2016 — CHANGES_REQUESTED** [review]
Add description and show `modify`.

**Comment by DaiYuzeng** [comment]
## Test steps
1. Go to Setting Page
2. Click `Edit Config` on harvester-monitoring section
3. Edit configuration
4. Succeed save
5. harvester-monitoring section will display correct JSON string
6. harvester-monitoring section on Setting Page will display modified status if you modified the configuration

**Review by WuJun2016 — APPROVED** [review]
LGTM

**Comment by w13915984028** [comment]
@guangbochen 

One doubt: the disk size seems to be limited to a dedicated partition of 50G now, does it make sense to change it here?
Not sure if user change it to a unavailable size, what will happen.

image

**Pull request merged**